src/StorageSync/StorageSync/CloudEndpoint/NewCloudEndpointCommand.cs

```diff
@@ -209,7 +210,7 @@ public override void ExecuteCmdlet()
                 var createParameters = new CloudEndpointCreateParameters()
                 {
                     StorageAccountResourceId = StorageAccountResourceId,
-                    StorageAccountShareName = AzureFileShareName,
+                    AzureFileShareName = AzureFileShareName,
```

Comment:
Why are you setting AzureFileShareName to itself?

Reply:
The "AzureFileShareName" on the right is a property of the NewCloudEndpointCommand class, while the "AzureFileShareName" on the left is a property of the CloudEndpointCreateParameters object being instanciated here. I could add "this" to the "AzureFileShareName" on the right side to highlight the difference, but it would be inconsistent with how the other properties are referenced, so I'm ok leaving it like this.